docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -220,11 +228,39 @@ to the following:
 
 **3.** Run your production build using a tool like [serve](https://www.npmjs.com/package/serve) (`serve -s build`) and verify the `<MapView>` component renders without error.
 
+### CRA 4: Can’t import the named export ‘Amplify’ from non EcmaScript module (only default export is available)
+
+Create React App v4 is not officially supported by Amplify UI. When you use it, you may get the following error
+
+```bash
+ ./node_modules/@aws-amplify/ui-react/dist/esm/components/..\/\*.mjs
+
+ Can’t import the named export ‘Amplify’ from non EcmaScript module (only default export is available)
+```
+
+To resolve the error, you may either (1) upgrade to Create React App version 5 ([Migration Guide](https://github.com/facebook/create-react-app/blob/main/CHANGELOG.md#migrating-from-40x-to-500)), or (2) to over-write the webpack config by using tools like [React App Rewired](https://github.com/timarney/react-app-rewired), [Craco](https://craco.js.org/docs/configuration/webpack/), using the following line:
```

Comment:
nit: `over-write` -> `overwrite`

Reply:
Done in f83847aba81caa942990d79ed350690dbf9ee945